lib/api/src/externals/function.rs

```diff
@@ -130,6 +138,9 @@ impl Function {
         F: Fn(&Env, &[Val]) -> Result<Vec<Val>, RuntimeError> + 'static,
         Env: Sized + 'static,
     {
+        if std::mem::size_of::<F>() != 0 {
+            Self::closures_unsupported_panic();
+        }
```

Comment:
I believe normal functions (not native), should still work. We should only disable native ones

Reply:
Oh, makes sense!  I've updated it to reflect that now